src/olympia/editors/tests/test_views.py

```diff
@@ -2320,10 +2320,10 @@ def test_no_versions(self):
 
         self.version.delete()
 
-        assert self.client.get(self.url).status_code == 404
+        assert self.client.get(self.url).status_code == 200
         response = self.client.post(self.url, {'action': 'info',
                                                'comments': 'hello sailor'})
-        assert response.status_code == 404
+        assert response.status_code == 302
```

Comment:
If we check for a redirect I'd be best to check we're redirecting to the correct `response['Location']` as well to avoid unwanted behavior.